tests/mechanic/launcher_test.py

```diff
@@ -93,6 +99,116 @@ def info(self, *args, **kwargs):
         return self._info
 
 
+logging.basicConfig(level=logging.DEBUG)
+HOME_DIR = os.path.expanduser("~")
+
+
+def create_config():
+    cfg = config.Config()
+    # collect some mandatory config here
+    cfg.add(config.Scope.application, "node", "root.dir", HOME_DIR + "/.rally/benchmarks")
```

Comment:
We'd usually use `os.path.join(HOME_DIR, ".rally", "benchmarks")` in such cases.